go.mod

```diff
@@ -532,34 +551,35 @@ replace (
 	k8s.io/component-base => ./staging/src/k8s.io/component-base
 	k8s.io/cri-api => ./staging/src/k8s.io/cri-api
 	k8s.io/csi-translation-lib => ./staging/src/k8s.io/csi-translation-lib
-	k8s.io/gengo => k8s.io/gengo v0.0.0-20200205140755-e0e292d8aa12
+	k8s.io/gengo => k8s.io/gengo v0.0.0-20200413195148-3a45101e95ac
 	k8s.io/heapster => k8s.io/heapster v1.2.0-beta.1
 	k8s.io/klog => k8s.io/klog v1.0.0
```

Comment:
isn't it problematic for us to still have things in our tree that use klog/v1?

Reply:
which version of klog do the CLI flags bind to?

Reply:
Looks like i may be missing a commit here. let me check

Reply:
klog is pretty pervasive, I expect we'll need a verify check that prevents reintroduction of klog/v1 usage. Will this also become a change all our library consumers have to react to in 1.19 (sweep their own code-bases and all their dependencies to unify on klog/v2)?

Reply:
so we no longer have klog v1 in our vendor/ with the latest updates i made in this PR

we will have to think through this a bit. folks using us as a library (and currently using klog v1 may have a way to get things to work). I have to try it out and add an example/doc somewhere